rxdogtag/src/main/resources/META-INF/proguard/rxdogtag.pro

```diff
@@ -1,3 +1,3 @@
 # We keep these in order for DogTagObservers to properly work and resolve entries in each of these packages
--keepnames class com.uber.rxdogtag.**
--keepnames class io.reactivex.**
+-keeppackagenames class com.uber.rxdogtag**
```

Comment:
You'll have to remove the `class`, otherwise it will cause an error.

Reply:
My bad. Applied!